dockerfile/cuda11.1.1.dockerfile

```diff
@@ -83,9 +83,8 @@ ENV PATH="${PATH}:/usr/local/cmake/bin:/usr/local/nccl-tests/build" \
     LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib" \
     SB_HOME="/opt/superbench"
 
-ARG SB_VERSION="main"
-RUN git clone -b ${SB_VERSION} https://github.com/microsoft/superbenchmark ${SB_HOME} && \
-    cd ${SB_HOME} && \
-    python3 -m pip install .[torch]
-
 WORKDIR ${SB_HOME}
+ADD . .
+
+RUN cd ${SB_HOME} && \
+    python3 -m pip install .[torch]
```

Comment:
Just as ever said, we should only install torchvision and transformers pacakges, and using .[torch] will change the torch version. Can we just remove torch requirement from .[torch] or add a new require term like .[torch_pkg]? Since .[torch] is used for test.dockerfile which has no torch environment.